### PR TITLE
Modify setting message to match options

### LINKED ISSUE #84978: Text Editor->Find options invalid
`Controls whether the find operation is carried out on selected text or the entire file in the editor.`
this is not a yes/no question, so the drop down doesn't make any sense.

image

### PR DESCRIPTION
**Pull request by jzyrobert**
This PR fixes #84978

Old: `Controls whether the find operation is carried out on selected text or the entire file in the editor.`
New: `Controls when the find operation is carried out on selected text instead of the entire file in the editor.`

The old message was intended for a boolean setting and it has changed since to a context based setting.

### CODE REVIEW

**Review by diguifi — APPROVED** [review]

**Comment by rebornix** [comment]
@sbatten can you get your help on this one?

File: src/vs/editor/common/config/editorOptions.ts [hunk 1]
```diff
@@ -1196,7 +1196,7 @@ class EditorFind extends BaseEditorOption<EditorOption.find, EditorFindOptions>
 						nls.localize('editor.find.autoFindInSelection.always', 'Always turn on Find in selection automatically'),
 						nls.localize('editor.find.autoFindInSelection.multiline', 'Turn on Find in selection automatically when multiple lines of content are selected.')
 					],
-					description: nls.localize('find.autoFindInSelection', "Controls whether the find operation is carried out on selected text or the entire file in the editor.")
+					description: nls.localize('find.autoFindInSelection', "Controls when the find operation is carried out on selected text instead of the entire file in the editor.")
```
Comment (sbatten) [inline src/vs/editor/common/config/editorOptions.ts]: I think the old structure is a bit confusing. I think we should update it to say "Controls the condition for turning on find in selection automatically."

**Review by ViniciusFXavier — APPROVED** [review]
:+1:

**Comment by jzyrobert** [comment]
@sbatten I have changed it to your suggestion.

**Review by sbatten — APPROVED** [review]

**Comment by jzyrobert** [comment]
@sbatten Realised this is still open. Still worth merging?

**Comment by rebornix** [comment]
@jzyrobert thanks for your help, it's great!

**Pull request merged**